scripts/build.py

```diff
@@ -191,27 +167,16 @@
     help='Build webpack with source maps.')
 
 
-def generate_app_yaml(deploy_mode=False, maintenance_mode=False):
+def generate_app_yaml(deploy_mode=False):
```

Comment:
Could you explain why this changed? Not sure I understand.

Reply:
We generate `app.yaml` from `app_dev.yaml`.
During generation, if maintenance mode is enabled, `generate_app_yaml`  function replaces the page's template with maintenance page's template. So, if a user visit that static page, he is served with maintenance page.
I have removed this because, they are no static pages in app_dev.yaml now and there will not be any in future.
They will be registered with angular rotuer now.